diff --git a/../st/config.h b/config.h
index 6f05dce..c894d32 100644
--- a/../st/config.h
+++ b/config.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Mono:pixelsize=18:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -96,24 +96,24 @@ unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+    "#262626", /* black   */
+    "#ff0000", /* red     */
+    "#5faf5f", /* green   */
+    "#d7af5f", /* yellow  */
+    "#005faf", /* blue    */
+    "#af00af", /* magenta */
+    "#005f5f", /* cyan    */
+    "#afafaf", /* white   */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+    "#5f5f5f", /* black   */
+    "#ff5f87", /* red     */
+    "#87d787", /* green   */
+    "#d7af87", /* yellow  */
+    "#00afff", /* blue    */
+    "#d75fd7", /* magenta */
+    "#00afd7", /* cyan    */
+    "#dadada", /* white   */
 
 	[255] = 0,
 
@@ -174,7 +174,7 @@ static uint forcemousemod = ShiftMask;
  */
 static MouseShortcut mshortcuts[] = {
 	/* mask                 button   function        argument       release */
-	{ XK_ANY_MOD,           Button2, selpaste,       {.i = 0},      1 },
+	{ XK_ANY_MOD,           Button3, clippaste,       {.i = 0} },
 	{ ShiftMask,            Button4, ttysend,        {.s = "\033[5;2~"} },
 	{ XK_ANY_MOD,           Button4, ttysend,        {.s = "\031"} },
 	{ ShiftMask,            Button5, ttysend,        {.s = "\033[6;2~"} },
